### PR TITLE
Replicas

### PR DESCRIPTION
**Pull request by Raruto**
- fix migration `0002_auto_20231213_1007.py`
- remove `x` attribute for trace replicas in json response (`QpsTimeseriesPlotDataApiView`)
- make use of `initConfig.layers` in client `plugin.js`

### CODE REVIEW

**Comment by wlorenzetti** [comment]
@Raruto can you work on this repository? https://github.com/Raruto/g3w-admin-ps-timeseries/tree/replicas use your repository get more time. In this way I can't push on this branch.

**Comment by wlorenzetti** [comment]
@Raruto do you have permissions on this repository

**Comment by Raruto** [comment]
> do you have permissions on this repository?

No, currently you have to add me on every new repo.

> I can't push on this branch.

You're probably trying with a wrong branch. By default maintainers always have access to PR code proposed by external people:

image

... otherwise it would be almost impossible to collaborate on open source code here on github 😢

**Comment by wlorenzetti** [comment]
@Raruto I'm going to merge it, I'll open another branch for g3w-admin serving plot data, ok?

**Comment by Raruto** [comment]
> I'm going to merge it

Ok, anyway I don't think I need to add anything else.

Or at least, as long as you don't need apply any breaking changes to server response.

**Comment by wlorenzetti** [comment]
@Raruto only one change:  could you add the project_pk to *qps-timeseries-api-plot-data* client side as I add to this commit? https://github.com/g3w-suite/g3w-admin-ps-timeseries/commit/306d0dce09a304eb3c466b1ff44312b1ec36aca8

File: qps_timeseries/static/qps_timeseries/js/plugin.js [hunk 1]
```diff
@@ -81,7 +81,7 @@
                       className: pid,
                     })
                     .on("shown.bs.modal", async function() {
-                      const { data, layout, config } = await (await fetch(initConfig.baseurl + 'qps_timeseries/api/plot/' + layer.id + '/' + feature.attributes[G3W_FID])).json();
+                      const { data, layout, config } = await (await fetch(initConfig.baseurl + 'qps_timeseries/api/plot/'+ initConfig.group.id +  '/' + layer.id + '/' + feature.attributes[G3W_FID])).json();
```
Comment (wlorenzetti) [inline qps_timeseries/static/qps_timeseries/js/plugin.js]: @raruto it sound not correct, I aspected to find a 'project' not a 'group'

**Pull request merged**